src/fu-jcat-gnutls-mldsa87-engine.c

```
@@ -0,0 +1,514 @@
+#include "config.h"
+
+#include "fu-jcat-engine.h"
+#include "fu-jcat-mldsa87-engine.h"
+#include "fu-jcat-pkcs7-common.h"
+
+struct _FuJcatGnutlsMldsa87Engine {
+	FuJcatEngine parent_instance;
+	gnutls_x509_trust_list_t tl;
+};
+
+G_DEFINE_TYPE(FuJcatGnutlsMldsa87Engine, fu_jcat_gnutls_mldsa87_engine, FU_TYPE_JCAT_ENGINE)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If G_DEFINE_TYPE is a macro then please configure it.

(unknownMacro)